src/mumble/PulseAudio.cpp

```diff
@@ -118,6 +118,14 @@ PulseAudioSystem::PulseAudioSystem() {
 	m_pulseAudio.threaded_mainloop_start(pam);
 
 	bRunning = true;
+
+	std::unique_lock< std::mutex > guard(m_initLock);
+	if (!m_initialized) {
+		// The mutex is atomically released as soon as this thread starts waiting and will be
+		// re-aquired when waking up.
+		// Spurious wake-ups are avoided by checking m_initialized in the given predicate
```

Comment:
"predicate"? You mean "context"?

Reply:
No - On https://en.cppreference.com/w/cpp/thread/condition_variable/wait it is called "predicate" so that is what I used here :shrug: